test/functional/basic/iwdp-e2e-specs.js

```diff
@@ -0,0 +1,39 @@
+// transpile:mocha
+
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { SAFARI_CAPS } from '../desired';
+import { initSession, deleteSession, MOCHA_TIMEOUT } from '../helpers/session';
+import B from 'bluebird';
+
+chai.should();
+chai.use(chaiAsPromised);
+
+if (process.env.REAL_DEVICE) {
+  describe('ios_webkit_debug_proxy: ', function () {
+    this.timeout(MOCHA_TIMEOUT);
+    let caps, driver;
+
+    beforeEach(async () => {
+      caps = Object.assign(SAFARI_CAPS);
+      caps.startIWDP = true;
+    });
+
+    afterEach(async () => {
+      await deleteSession();
+      await B.delay(2000);
```

Comment:
I saw a delay be put in other places that did similar tests. I'm going to try reducing it back to 500 ms though.